Dockerfile.windows

```
@@ -163,6 +164,10 @@ RUN `
   Write-Host INFO: Expanding compiler 3 of 3...; `
   Expand-Archive C:\binutils.zip -DestinationPath C:\gcc -Force;
 
+RUN `
+  Write-Host 'INFO: Including busybox ...'; `
+  c:\busybox\busybox.exe --list ^|ForEach-Object {$nul = cmd /c mklink C:\busybox\$_.exe busybox.exe}
```

Nit; more consistent to use uppercase `C:\`

```suggestion
  C:\busybox\busybox.exe --list ^|ForEach-Object {$nul = cmd /c mklink C:\busybox\$_.exe busybox.exe}
```